src/App/components/ActionLink/index.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const ActionLink = ({linkText, badge, smallText}) => (
```

Comment:
Usage of small text will be implemented in [DG-384](https://payexjira.atlassian.net/secure/RapidBoard.jspa?rapidView=80&projectKey=DG&modal=detail&selectedIssue=DG-384).

Reply:
Yes, probably :)